pyttsx3/drivers/espeak.py

```diff
@@ -65,9 +69,14 @@ def destroy():
         _espeak.SetSynthCallback(None)
 
     def stop(self):
-        if _espeak.IsPlaying():
-            self._stopping = True
-            _espeak.Cancel()
+        if not self._stopping:
+            logging.debug("[DEBUG] EspeakDriver.stop called")
```

Comment:
Take advantage of line 13 above.
```suggestion
            logger.debug("[DEBUG] EspeakDriver.stop called")
```